components/select_team/index.ts

```diff
@@ -47,6 +50,7 @@ function mapStateToProps(state: GlobalState) {
         siteURL: config.SiteURL,
         totalTeamsCount: state.entities.teams.totalCount || 0,
         isCloud,
+        isFreeTrial,
```

Comment:
This is for consistence across restricted features, not using limits during trial.

Reply:
Not sure I understand what you mean here - during trial they have no limits so checking the limits at all times should be enough

Reply:
Limits are still hanging when trial is started, this just skips the flakiness in the service data for this use case.